pkg/kubevip/config_manager.go

```diff
@@ -205,6 +206,10 @@ func isValidInterface(iface string) error {
 		return fmt.Errorf("get %s failed, error: %w", iface, err)
 	}
 
+	if l.Attrs().Flags&net.FlagLoopback == net.FlagLoopback {
```

Comment:
I think a linter would add some space here.

Reply:
golangci-lint, go vet, and go fmt aren't suggesting anything. What are you thinking would be better?

Reply:
golangci-lint, go vet, and go fmt aren't suggesting anything. What are you thinking would be better?

Reply:
https://github.com/kube-vip/kube-vip/runs/4764985605?check_suite_focus=true#step:6:116

apparently not :D

Reply:
I stand corrected - I tried `gofumpt -l -w .` and `golangci-lint run ./... --fix` (with the enabled settings for this project, also with the tinkerbell/tink settings). No change.